recipes/petsc/meta.yaml

```diff
@@ -21,7 +21,7 @@ build:
 
 requirements:
   build:
-    - python
+    - python <3
```

Comment:
Generally we enforce this with the skip preprocessor - so above where you skip windows we would also skip python<3 -> `skip: true  # [win or py2k]`